subprojects/extTest/src/test/java/org/mockitousage/plugins/donotmockenforcer/DoNotmockEnforcerTest.java

```diff
@@ -41,4 +79,10 @@ static class NotMockable {}
 
     @DoNotMock
     static class NotMockableButSpecialCased {}
+
+    static class StaticallyNotMockable {}
```

Comment:
Can we also add a regression test that creates a regular mock for this type and asserts that it can still be mocked as before?

Reply:
Sure thing. Will add a special case for it.